_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -0,0 +1,114 @@
+---
+id: aaa1bf
+name: auto-play audio does not exceed 3 seconds
+rule_type: atomic
+description: |
+  auto-play audio content does not output audio for more than 3 seconds
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML [`audio`](https://html.spec.whatwg.org/multipage/media.html#the-audio-element), [`video`](https://html.spec.whatwg.org/multipage/media.html#the-video-element) or [`source`](https://html.spec.whatwg.org/multipage/embedded-content.html#the-source-element) elements, with a [`src`](https://html.spec.whatwg.org/multipage/media.html#attr-media-src) attribute referencing content with a duration of more than 3 seconds, that has an [`autoplay`](https://html.spec.whatwg.org/multipage/media.html#attr-media-autoplay) attribute equal to `true`, and that has both [`paused`](https://html.spec.whatwg.org/multipage/media.html#dom-media-paused) and [`muted`](https://html.spec.whatwg.org/multipage/media.html#attr-media-muted) attributes equal to `false`.
+
+## Expectation
+
+For each test target the total audio output does not last more than 3 seconds.
```

Comment:
It might be good to put in a note that says this also applies to silent videos. That's the most convincing scenario for this rule IMO, and it's not immediately obvious that's what it does.

Reply:
@WilcoFiers I'm not sure I understand your point. If a video is silent, it doesn't have any audio output - this rule is for no auto play audio.

Reply:
I'm assuming you are referring to a silent auto-play video... which isn't covered by this rule which maps to 1.4.2 which is specifically about "Audio control".

There is no equivalent "Video control" SC... a bit of a gap in the guidelines perhaps.

Reply:
There is nothing in the applicability that excludes silent videos, so the rule does apply to silent videos. If that's not the intention, you'll need to change the applicability.